packages/common/sentry/src/common.ts

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { addBreadcrumb as naturalAddBreadcrumb, captureException as naturalCaptureException } from '@sentry/browser';
```

Comment:
what's wrong with the node version?

Reply:
oof, i couldn't understand why the code was duplicated after comparing the two methods but didn't think to compare the import.

will revert this bit